lib/hooks/userspace/hook-express.js

```diff
@@ -83,7 +83,7 @@ function startRootSpanForRequest(req, traceHeader) {
   // we use the path part of the url as the span name and add the full
   // url as a label
   var rootContext = agent.createRootSpanData(req.originalUrl, traceId,
-    parentSpanId);
+    parentSpanId, 3); // Skip 3 frames
```

Comment:
This mechanism will be brittle when we change something in how `createRootSpanData`. `createRootSpanData` should have an idea how how many frames it should skip regardless, and it could accept an argument that is the number of frames to skip *in addition* so that the caller of `createRootSpanData` can skip additional frames.

It would be good to reduce the unnecessary tight coupling here.